src/win32/fs.cpp

```diff
@@ -1656,7 +1656,7 @@ DirNotify* WinFileSystemAccess::newdirnotify(LocalPath& localpath, LocalPath& ig
     return new WinDirNotify(localpath, ignore, this, waiter);
 }
 
-bool WinFileSystemAccess::issyncsupported(string *localpath, bool *isnetwork, SyncError *syncError)
+bool WinFileSystemAccess::issyncsupported(LocalPath& localpathArg, bool *isnetwork, SyncError *syncError)
```

Comment:
In order to avoid this kind of name clashes, we should use a prefix `aXYZ` when needed.
```suggestion
bool WinFileSystemAccess::issyncsupported(LocalPath& aLocalpath, bool *isnetwork, SyncError *syncError)
```